upstream-tools/replacements.json

```diff
@@ -698,6 +706,10 @@
     }
   ],
   "website/docs/r/connect_contact_flow.html.markdown": [
+    {
```

Comment:
This seems to duplicate the entry immediately after it -- am I misinterpreting what's going on?

Reply:
It varies by the comma it seems - I've just pulled these in from Thomas. No harm having near-duplicates, but the latter rule could probably be removed with this new one.

Reply:
Well spotted! (or: poorly spotted by me!) OK so the one that comes first does not have commas in the match or the replacement text -- I guess they were removed in the upstream. The second one presumably won't match because the match text doesn't match.

Reply:
Should we expect that second one to be removed as unused in the commit after this one?

Reply:
Ah! There are two very similar snippets, one has a comma and one doesn't:
https://github.com/pulumi/terraform-provider-aws/blob/fc7794404f569e5ee9f1c80963daee017ccb8f0f/website/docs/r/connect_contact_flow.html.markdown?plain=1#L59
https://github.com/pulumi/terraform-provider-aws/blob/fc7794404f569e5ee9f1c80963daee017ccb8f0f/website/docs/r/connect_contact_flow.html.markdown?plain=1#L85
As you were.